MapboxCoreNavigation/RouteController.swift

```diff
@@ -583,7 +583,10 @@ extension RouteController: CLLocationManagerDelegate {
         let newLocation = CLLocation(latitude: locationInfrontOfUser.latitude, longitude: locationInfrontOfUser.longitude)
         let radius = max(reroutingTolerance, location.horizontalAccuracy + RouteControllerUserLocationSnappingDistance)
         let isCloseToCurrentStep = newLocation.isWithin(radius, of: routeProgress.currentLegProgress.currentStep)
-
+        
+        if isCloseToCurrentStep {
```

Comment:
Nit: I generally like to see conditional branches that early-exit as `guard` statements, but that's largely a style thing.